Documentation/guides/messages/xa0125.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Xamarin.Android warning XA0125
+description: XA0125 warning code
+ms.date: 09/28/2020
+---
+# Xamarin.Android warning XA0125
+
+## Example messages
+
+```
+warning XA0125: 'AndroidApp1.pdb' is using a deprecated debug information level. Set the debugging information to Portable in the Visual Studio project property pages or edit the project file in a text editor and set the 'DebugType' MSBuild property to 'portable' to use the newer, cross-platform debug information level.
+```
+
+## Issue
+
+Using symbol files with the `.mdb` extension or
+`<DebugType>full</DebugType>` is deprecated. Using non-portable symbol
+files with the `.pdb` extension or `<DebugType>pdbonly</DebugType>` is
+deprecated. This also includes NuGet packages or any library that
+distributes `.mdb` or non-portable `.pdb` files.
+
+## Solution
+
+`full` or `pdbonly` symbol files are a Windows-specific format and
+`<DebugType>portable</DebugType>` should be used going forward. .NET 6
+and higher will not support the older formats. Notify the library
+author if you are encountering the `XA0125` warning from a NuGet package.
+
+In Visual Studio, go to `Properties > Build > Advanced...` and change
+`Debugging Information` to `Portable`.
```

Comment:
To align with the [docs guidelines](https://github.com/MicrosoftDocs/xamarin-docs/blob/c7d82b0684b1c26cdcbc16766b636894a64a80fd/contributing-guidelines/template.md#text-styling), this can use `**strong**` for UI elements:

```markdown
In Visual Studio, go to **Properties > Build > Advanced** in the project
property pages and change **Debugging information** to **Portable**.
```

And Visual Studio for Mac can be added:

```markdown
In Visual Studio for Mac, go to **Build > Compiler > Debug information**
in the project property pages and change **Debug information** to
**Portable**.
```